### PR TITLE
Install :)

### PR DESCRIPTION
**Pull request by terrillo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # postcss-responsify
 [PostCSS](https://github.com/postcss/postcss) plugin to create responsive classes with the `@responsive` rule.
 
+**Install**
+```
+npm install postcss-responsify --save
```
Comment (derek-duncan) [inline README.md]: We should save as a dev dependency. `npm install postcss-responsify --save-dev`

**Review by derek-duncan — CHANGES_REQUESTED** [review]
Noice. This is essential to _every_ npm package. Thanks!

**Comment by terrillo** [comment]
@derek-duncan updated

**Review by terrillo — COMMENTED** [review]
Ready for review

**Pull request merged**